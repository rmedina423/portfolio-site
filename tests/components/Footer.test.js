import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../src/js/components/Footer';

describe('Footer Component', function () {
  beforeEach(() => {
    this.footer = shallow(<Footer />);
  });

    test('should match snapshot', () => {
    expect(this.footer.getElement()).toMatchSnapshot();
  });
});