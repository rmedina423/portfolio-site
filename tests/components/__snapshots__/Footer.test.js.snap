// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match snapshot 1`] = `<footer />`;
